src/Symfony/Component/Validator/ConstraintValidator.php

```diff
@@ -88,6 +88,14 @@ protected function formatValue(mixed $value, int $format = 0): string
             return $value->format('Y-m-d H:i:s');
         }
 
+        if (\is_object($value) && $value instanceof \UnitEnum) {
```

Comment:
is_object check is not needed, instanceof in enough

Reply:
Correct, I'll remove it.